src/embind/embind.js

```diff
@@ -762,8 +762,13 @@ var LibraryEmbind = {
   },
 
   _embind_register_std_wstring__deps: [
-    '$readLatin1String', '$registerType',
-    '$simpleReadValueFromPointer'],
+    '$readLatin1String', '$registerType', '$simpleReadValueFromPointer'
```

Comment:
I think you can leave the tailing comma on the this line and on line 769. 

I find that doing this often makes diffs easier to read